kolibri/core/auth/management/commands/bulkimportusers.py

```diff
@@ -426,12 +434,12 @@ def csv_values_validation(self, reader, header_translation):
         )
         validator.add_check(
             "ENROLLED_IN",
-            value_length(100, allow_null=True),
+            value_length(100, allow_null=True, multiple=True),
```

Comment:
Should the first arg be `50` + `multiple=True`, meaning we allow multiple names of up to 50 characters?

Reply:
are you sure? In the first iterations of this command I was told classes could have up to 100 chars? Obviously I have not any problem in changing it, but I'd like to be sure this is the right limit.

Reply:
The `Collection` model does allow names up to 100, but the user interface limits classroom names to 50 characters.

Reply:
This is also true for Learner Groups

Reply:
Thanks for confirming!
I have done the change.